api/database/services.py

```diff
@@ -0,0 +1,144 @@
+import os
+from flask import Flask, request, jsonify
+from flask_pymongo import PyMongo
+import json
+
+def index():
+    """Display beacon info."""
+    beacon_info = {
+        "id": ".".join(reversed(request.host.split("."))),
+        "name": "Imaging beacon",
+        "type": {'group': f'{"test"}',
+                    'artifact': f'{"beacon"}',
+                    'version': f'{"0.0.0"}'},
+        "description": "bp test beacon",
+        "organization": {
+            "name": "",
+            "url": "",
+        },
+        "contactUrl": "",
+        "documentationUrl": "",
+        "createdAt": "",
+        "updatedAt": "",
+        "environment": "",
+        "version": "",
+    }
+    return beacon_info
+
+def getItems(db):
+    """List all db items."""
+    dbImages = db.images.find()
+    dbDataset = db.dataset.find()
+    dbSamples = list(db.sample.find())
+    for coll in db.list_collection_names():
+        print(coll)
```

Comment:
```suggestion
```
remove debug prints, or start using a logging module for debug logging

Reply:
example https://github.com/CSCfi/beacon-python/blob/master/beacon_api/utils/logging.py `LOG.debug(something)`